include/spdk/blob.h

```diff
@@ -1055,6 +1055,12 @@ void spdk_bs_iter_first(struct spdk_blob_store *bs,
 void spdk_bs_iter_next(struct spdk_blob_store *bs, struct spdk_blob *blob,
 		       spdk_blob_op_with_handle_complete cb_fn, void *cb_arg);
 
+/**
+ * Get the current blob's parent.
+ * \param blob The current blob.
+ * \return if parent blob exist send blob else NULL.
+ */
+struct spdk_blob *spdk_bs_iter_parent(struct spdk_blob *blob);
```

Comment:
Let's not use `iter` in the name because we are simply fetching the parent and not really iterating. How about `spdk_bs_get_parent_blob`

Reply:
The name is kept just to have uniformity on the name for other API provided from blob.
`spdk_bs_iter_next` and `spdk_bs_iter_first` are existing API. So idea is to keep the naming with the same convention.